external/brigand/README.md

```diff
@@ -0,0 +1,40 @@
+Brigand Meta-programming library
```

Comment:
Include the upstream commit hash this is based on somewhere.  Maybe in the commit message.

Reply:
I did this and also squashed in the commit hash at the end of the README, just so that it's hopefully easier to find for someone in the future.